bigtable/tests/unit/test_instance.py

```diff
@@ -897,6 +899,114 @@ def test_list_app_profiles(self):
         self.assertIsInstance(app_profile_2, AppProfile)
         self.assertEqual(app_profile_2.name, app_profile_name2)
 
+    def test_get_iam_policy(self):
+        from google.cloud.bigtable_admin_v2.gapic import (
+            bigtable_instance_admin_client)
+        from google.cloud.iam import Policy
+
+        instance_api = (
+            bigtable_instance_admin_client.BigtableInstanceAdminClient(
+                mock.Mock()))
+
+        credentials = _make_credentials()
+        client = self._make_client(project=self.PROJECT,
+                                   credentials=credentials, admin=True)
+        instance = self._make_one(self.INSTANCE_ID, client)
+
+        version = 1
+        etag = b'etag_v1'
+        bindings = [{'role': 'roles/owner',
+                     'members': ['serviceAccount:service_acc1@test.com',
+                                 'user:user1@test.com']}]
+        policy = {
+            'version': version,
+            'etag': etag,
+            'bindings': bindings
+        }
+        response_pb = policy_pb2.Policy(**policy)
```

Comment:
Rather than constructing a dict, and then passing it via `**`, it is clearer to use keyword arguments:

```python
        response_pb = policy_pb2.Policy(version=version, etag=etag, bindings=bindings)
```